File: FizzBuzz.cpp
/*
 * FizzBuzz.cpp
 * Andrew Nolan - acnolan
 */

#include <iostream>

using namespace std;

int main(int argc, char* argv[]){
	//print the numbers from 1 to 100
	//replace every number divisible by 3 with Fizz
	//replace every number divisible by 5 with Buzz
	//replace every number divisible by both with FizzBuzz
	for(int i=1; i<101; i++){
		int flag=0;
		if(i%3==0){
			cout<<"Fizz";
			flag=1;
		}
		if(i%5==0){
			cout<<"Buzz";
			flag=1;
		}
		if(!flag){
			cout<<i;
		}
		cout<<"\n";
	}
}

File: FizzBuzz.py
#FizzBuzz.py
#Andrew Nolan - acnolan

#print all numbers from 1 to 100
#if the number is divisible by 3 replace it with Fizz
#if the number is divisible by 5 replace it with Buzz
#if the number is divisible by both replace it with FizzBuzz
for x in range(1,101):
    flag = 0 
    if x % 3 == 0:
        print("Fizz", end='')
        flag=1
    if x % 5 == 0:
        print("Buzz", end='')
        flag=1
    if not flag:
        print(x, end='')
    print()
        

    